pkgs/servers/mail/postfix/default.nix

```diff
@@ -34,7 +34,7 @@ in stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ makeWrapper m4 ];
-  buildInputs = [ db openssl cyrus_sasl icu libnsl pcre ]
+  buildInputs = [ db openssl cyrus_sasl icu libnsl pcre cacert ]
```

Comment:
I don't think this is required since you specify it in the postfix configuration.

Reply:
Are you sure about that?